packages/react-experiments/src/components/FloatingSuggestionsComposite/FloatingSuggestionsList/FloatingSuggestionsList.tsx

```diff
@@ -7,6 +7,7 @@ import {
 } from './FloatingSuggestionsList.types';
 import { FloatingSuggestionsItemMemo } from '../FloatingSuggestionsItem/FloatingSuggestionsItem';
 import { getStyles } from './FloatingSuggestionsList.styles';
+import { ISuggestionsHeaderFooterProps, SuggestionsHeaderFooterItem } from '@fluentui/react/lib/FloatingPicker';
```

Comment:
I would not depend on FloatingPicker component, let's create the interfaces in FloatingSuggestionsComposite component, since eventually we would want FloatingPicker component to go away ( since it is inheritance based and we intend to not use it)